source/Tipster.cpp

```diff
@@ -62,15 +62,21 @@ Tipster::Tipster()
 
 	fTipsterTextView = new TipsterText();
 	fIcon = new BButton("iconview", "", new BMessage(OPEN_URL));
+	fIcon -> SetToolTip( B_TRANSLATE("Open the Tipster homepage") );
```

Comment:
That's not correct, the icon normally leads to "more information", most of the time the User Guide.